### PR TITLE
Flake8 configuration file

### PR DESCRIPTION
**Pull request by sshakndr**
Flake8 rules was moved to configuration file. Two useless invokes of flake in Github Actions are melded. Correct flake invoke is added to makefile.

### CODE REVIEW

**Comment by vanyauhalin** [comment]
@sshakndr, please, change the target branch from the `master` to `develop`.

**Comment by sshakndr** [comment]
> @sshakndr, please, change the target branch from the `master` to `develop`.

already done. sorry for this

File: web/documentserver-example/python/Makefile [hunk 1]
```diff
@@ -47,8 +47,7 @@ compose-prod: #  Up containers in a production environment.
 
 .PHONY: lint
 lint: #          Lint the source code for style and check for types.
-	@flake8
-	@mypy .
```
Comment (vanyauhalin) [inline web/documentserver-example/python/Makefile:51]: Why did you delete the `mypy` checker?

File: web/documentserver-example/python/.flake8 [hunk 2]
```diff
@@ -0,0 +1,4 @@
+[flake8]
+max-complexity = 15
+max-line-length = 120
+per-file-ignores = __init__.py:F4
```
Comment (vanyauhalin) [inline web/documentserver-example/python/.flake8:4]: Could you please explain to me what `per-file-ignores` means?

**Review by vanyauhalin — CHANGES_REQUESTED** [review]

File: web/documentserver-example/python/.flake8 (hunk 2 above)
Comment (sshakndr) [reply web/documentserver-example/python/.flake8:4]: > Could you please explain to me what `per-file-ignores` means?

`per-file-ignores = A:B` will ignore `B` rules in `A` file(s). F4 codes is about importing modules, but flake8 is not friendly to init files. For example it will always trow F401(unused import).

**Review by vanyauhalin — APPROVED** [review]
